frontend/src/css/modules/sidebar.scss

```diff
@@ -1,12 +1,13 @@
 .ev-sidebar {
     display: block;
-    height: 100vh;
+    height: calc(100vh - 120px);
```

Comment:
In new design it's not. Sidebar is not starting from topmost point. It has some top space which is equal to 120px here also this happens due to it's fixed position nature. Also in case of mobile we will have different sidebar which has full viewport height, so there is no need to remove 120px here. Getting?